pkg/sessions/persistence/ticket.go

```diff
@@ -90,6 +90,8 @@ func decodeTicket(encTicket string, cookieOpts *options.Cookie) (*ticket, error)
 	}, nil
 }
 
+var ErrCookieValidationFailed = errors.New("session ticket cookie failed validation")
```

Comment:
Please make this a const and move to the top of the file below the other type definitions.

Reply:
I moved it to the top of the file, but i cannot make it const. I get a error from `make` in that case:

```
$ make
GO111MODULE=on golangci-lint run
WARN [runner] The linter 'deadcode' is deprecated (since v1.49.0) due to: The owner seems to have abandoned the linter. Replaced by unused.
WARN [runner] The linter 'structcheck' is deprecated (since v1.49.0) due to: The owner seems to have abandoned the linter. Replaced by unused.
WARN [runner] The linter 'varcheck' is deprecated (since v1.49.0) due to: The owner seems to have abandoned the linter. Replaced by unused.
WARN [runner] The linter 'golint' is deprecated (since v1.41.0) due to: The repository of the linter has been archived by the owner. Replaced by revive.
pkg/sessions/session_store_test.go:13:2: could not import github.com/oauth2-proxy/oauth2-proxy/v7/pkg/sessions/persistence (-: # github.com/oauth2-proxy/oauth2-proxy/v7/pkg/sessions/persistence
pkg/sessions/persistence/ticket.go:46:35: errors.New("session ticket cookie failed validation") (value of type error) is not constant) (typecheck)
        "github.com/oauth2-proxy/oauth2-proxy/v7/pkg/sessions/persistence"
        ^
pkg/sessions/persistence/interfaces.go:1: : # github.com/oauth2-proxy/oauth2-proxy/v7/pkg/sessions/persistence [github.com/oauth2-proxy/oauth2-proxy/v7/pkg/sessions/persistence.test]
pkg/sessions/persistence/ticket.go:46:35: errors.New("session ticket cookie failed validation") (value of type error) is not constant (typecheck)
package persistence
pkg/sessions/redis/redis_store.go:14:2: could not import github.com/oauth2-proxy/oauth2-proxy/v7/pkg/sessions/persistence (-: # github.com/oauth2-proxy/oauth2-proxy/v7/pkg/sessions/persistence
pkg/sessions/persistence/ticket.go:46:35: errors.New("session ticket cookie failed validation") (value of type error) is not constant) (typecheck)
        "github.com/oauth2-proxy/oauth2-proxy/v7/pkg/sessions/persistence"
        ^
make: *** [Makefile:34: lint] Error 1
```

Reply:
@chrta-wmi my mistake you are right. Function calls cannot be used as consts.